src/serializer/visitors.js

```diff
@@ -116,6 +135,23 @@ export let ClosureRefReplacer = {
     }
   },
 
+  // TODO: handle FunctionDeclaration
+  FunctionExpression(path: BabelTraversePath, state: ClosureRefReplacerState) {
+    if (t.isProgram((path: any).parentPath.parentPath.node)) {
```

Comment:
Don't end with : any cast, add cast to type which is supposed to have .parentPath property. Or maybe the BabelTraversePath is missing a property declaration?

Reply:
Yes. BabelTraversePath is missing "parentPath" property flow declaration which I do not think I can modify. I will add a comment.

Reply:
Yes, you can modify. Just edit ./flow-typed/npm/babel-traverse_vx.x.x.js. If you are wondering where those files came from --- I scooped up some stuff from the web and heavily modified it. It's not authoritative, just good guidance!